test/ctia/test_helpers/core.clj

```diff
@@ -27,45 +27,57 @@
             [ctim.generators.common :as cgc]
             [flanders
              [spec :as fs]
-             [utils :as fu]]))
+             [utils :as fu]]
+            [puppetlabs.trapperkeeper.app :as app]))
 
 (def ^:dynamic ^:private *config-transformers* [])
 
 (defn get-service-map [app svc-kw]
```

Comment:
Now either real or stub TK apps can be floating around. This function is gross but centralizes a lot of changes during this transition.